Improvement: Corrected the link for Authorization.

Shubham82 opened the pull request:
This PR fixes the link of `Authorization` under the [API Access Extensions](https://kubernetes.io/docs/concepts/extend-kubernetes/#authorization) in the [Extending Kubernetes](https://kubernetes.io/docs/concepts/extend-kubernetes/) docs.

Fixes: #30429

Shubham82 commented:
Hi @cheftako @lavalamp @bradtopol
PTAL!

seokho-son commented:
@Shubham82 Thanks !

/lgtm

Shubham82 commented:
Thanks, @seokho-son  for taking look at this.

mehabhalodiya reviewed (commented):
LGTM!
Thanks, @Shubham82 for improvement :)

Shubham82 commented:
> LGTM! Thanks, @Shubham82 for improvement :)

You're welcome! @mehabhalodiya

kbhawkey commented on content/en/docs/concepts/extend-kubernetes/_index.md:
```diff
@@ -145,7 +145,7 @@ Kubernetes provides several built-in authentication methods, and an [Authenticat
 ### Authorization
 
-[Authorization](/docs/reference/access-authn-authz/webhook/) determines whether specific users can read, write, and do other operations on API resources. It works at the level of whole resources -- it doesn't discriminate based on arbitrary object fields. If the built-in authorization options don't meet your needs, and [Authorization webhook](/docs/reference/access-authn-authz/webhook/) allows ca ...
+[Authorization](/docs/reference/access-authn-authz/authorization/) determines whether specific users can read, write, and do other operations on API resources. It works at the level of whole resources -- it doesn't discriminate based on arbitrary object fields. If the built-in authorization options don't meet your needs, and [Authorization webhook](/docs/reference/access-authn-authz/webhook/) all ...
```
HI @Shubham82 . The link update looks great.
What do you think about removing the word, "and" before the second link text (Authorization webhook)?

Shubham82 replied:
Hi @kbhawkey, Thanks for your suggestion
It seems good to me, I will update it soon.
Have you any suggestions other than that?

kbhawkey commented:
Page preview:
https://deploy-preview-30430--kubernetes-io-main-staging.netlify.app/docs/concepts/extend-kubernetes/

Shubham82 commented:
Hi @kbhawkey 
> What do you think about removing the word, "and" before the second link text (Authorization webhook)?

Addressed the comments, PTAL!

jihoon-seo commented:
/lgtm

RinkiyaKeDad reviewed (approved):
LGTM too!

RinkiyaKeDad commented:
@Shubham82 you'll have to sign the CLA before we can merge this 🙂

Shubham82 commented:
Hi, @RinkiyaKeDad  I have already signed the CLA. This EasyCLA is optional. 
Please refer to this: https://github.com/kubernetes/test-infra/pull/24381

Shubham82 commented:
HI @seokho-son @jihoon-seo
If everything seems good then Could you please approve this PR?
Thanks

kbhawkey commented:
/label tide/merge-method-squash

kbhawkey commented:
Hi, @Shubham82 .
The changes look good.
I see that the CNCF CLA label has been applied. However, do you need to click the `EastCLA` to authorize?
Thanks.

Shubham82 commented:
Hi @kbhawkey 

> However, do you need to click the EasyCLA to authorize (and enable all CI/CD checks)?

I don't think I need to authorize EasyCLA to merge this PR. It is an optional thing.

> Another idea, could you rebase to a single commit (one of the commits is attached to the EasyCLA)?

yes, I will squash it to a single commit.

Shubham82 commented:
Hi @kbhawkey

PTAL!

tengqm commented:
@Shubham82 CLA signing is mandatory, not optional.

Shubham82 commented:
Hi @tengqm
I already signed the CLA. This EasyCLA is a different thing that is optional and ignored by Kubernetes as of now.
Ref: kubernetes/test-infra#24381

Shubham82 commented:
/easycla

RA489 commented:
https://github.com/kubernetes/org/issues/3068
/easycla

RA489 commented:
/retest

Shubham82 commented:
Hi, @kbhawkey @sftim @tengqm  Looking for approval level.

mrbobbytables commented:
EasyCLA is in non-blocking mode to surface various issues between the old system and the new one, it's okay that it's reporting an error as long as its reported in the tracking thread (it is).
See the announcement about it here: https://groups.google.com/g/kubernetes-dev/c/-mdBNXLYZ_Y/m/Bp7B4W7IAAAJ
Feedback thread here: https://github.com/kubernetes/org/issues/3068

sftim commented:
/remove-label tide/merge-method-squash
/approve
